index.bs

```diff
@@ -155,6 +155,33 @@ the user agent MUST run the following steps:
     steps.
 6. Render |pipVideo| frames in the |video|.
 
+## Disable Picture in Picture
+
+Some pages may want to disable Picture In Picture for a video element. To
+support this, a new {{disablePictureInPicture}} attribute is added to the list
+of content attributes for video elements.
+
+A corresponding {{disablePictureInPicture}} IDL attribute which reflects the
+value of element’s {{disablePictureInPicture}} content attribute is added to
+the {{HTMLVideoElement}} interface. The {{disablePictureInPicture}} IDL attribute
+MUST reflect the content attribute of the same name.
+
+If the {{disablePictureInPicture}} attribute is present on the video element,
+the user agent SHOULD NOT play the video element in picture in picture or
+present any UI to do so.
+
+When the {{disablePictureInPicture}} attribute is added to a |video| element,
+the user agent SHOULD run these steps:
+
+1. Reject any pending promises returned by the {{requestPictureInPicture()}}
```

Comment:
Shouldn't you add steps in the enter picture in picture algorithm to reject if the attribute is set too?

Reply:
Fixed in https://github.com/WICG/picture-in-picture/pull/28/commits/04ce8d3084193ab3c7ff7d23808eee0c59450cdb